openedx_learning/apps/authoring/publishing/api.py

```diff
@@ -1211,12 +1212,17 @@ def get_containers(
     Args:
         learning_package_id: The primary key of the learning package
         container_cls: The subclass of Container to use, if applicable
+        include_deleted: If True, include deleted containers in the result.
```

Comment:
```suggestion
        include_deleted: If True, include deleted containers (with no draft version) in the result.
```
I think we should just clarify that we mean "draft deleted", since deleting the draft and deleting the published are separate actions.